### DISCUSSION TITLE
Add Sunset Beach as background

### DISCUSSION THREAD

**Discussion by hosttry** [BuddiesOfBudgie/budgie-backgrounds#8] [Background Suggestions]
Image Information
- Autor: Ian Livesey
- License: CC0 1.0 Universal (CC0 1.0), Public Domain Dedication
- Source: https://stocksnap.io/photo/sunset-beach-1FU4KJBQSZ

**Comment by serebit** [comment]
I don't see the license on that page, where can I find it?

**Reply by hosttry** [reply]
I totally forgot about it
here it is:
https://stocksnap.io/license
